Versioning.md

```diff
@@ -1,32 +1,37 @@
 # Guidance on Coyote versioning
+
 #### v1.0.0
 
 The Coyote framework versioning follows [Semantic Versioning 2.0.0](https://semver.org/) and has the pattern of MAJOR.MINOR.PATCH.
 
-1) MAJOR version when you make incompatible API changes,
-2) MINOR version when you add functionality in a backwards compatible manner, and
-3) PATCH version when you make backwards compatible bug fixes.
+1. MAJOR version when you make incompatible API changes,
+2. MINOR version when you add functionality in a backwards compatible manner, and
+3. PATCH version when you make backwards compatible bug fixes.
 
 We adopt everything listed in [Semantic Versioning 2.0.0](https://semver.org/)
 but to summarize the major points here:
 
 **Incrementing the MAJOR version number** should be done when:
- * a major new feature is added to the Coyote framework or tools and is showcased by a new tutorial demonstrating this feature
- * a breaking change has been made to Coyote API or serialization format, or
- tool command line arguments.
+
+- a major new feature is added to the Coyote framework or tools and is showcased by a new tutorial demonstrating this feature
+- a breaking change has been made to Coyote API or serialization format, or
+  tool command line arguments.
 
 **Incrementing the MINOR version number** should be done when:
- * new features are added to the Coyote API or tools that are backwards
- compatible
+
+- new features are added to the Coyote API or tools that are backwards
+  compatible
 
 **Incrementing the PATCH version number** should be done when
- * anything else changes in the the Coyote framework.
+
```

Comment:
Same here, please revert this to not have a new line added. Makes no difference to the output.